devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -71,6 +73,9 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     bool m_streamJointStates{false};
     bool m_streamMotorPWM{false};
     bool m_streamPIDs{false};
+    bool m_streamWrenches{false};
+    bool m_streamFTs{false};
+    bool m_streamInertials{false};
```

Comment:
Those parameters are not used 😄

Reply:
Now yes :)

Reply:
In theory, you do need them. Indeed if wrench are streamed the associated map is empty

Reply:
Indeed I added the check to fill the map

https://github.com/isorrentino/bipedal-locomotion-framework/blob/55e8b428c2dddc4bf69f2a0df6e0cde028994349/devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp#L247-L332

Reply:
Got your point. Changed again the code.